sormas-api/src/main/java/de/symeda/sormas/api/person/ArmedForcesRelationType.java

```diff
@@ -0,0 +1,18 @@
+package de.symeda.sormas.api.person;
+
+import de.symeda.sormas.api.i18n.I18nProperties;
+
+public enum ArmedForcesRelationType {
+
+	NOT_COLLECTED,
```

Comment:
Not null value can be mapped to SurvNet as `NOT_COLLECTED`, so it's not needed in the enum.
> Holger told me for an other enum: " so `nicht ermittelbar` should be unknown, and `nicht erhoben` should be null."